README.md

```diff
@@ -4,6 +4,7 @@
 `imgix-java` is a client library for generating image URLs with [imgix](https://www.imgix.com/).
 
 [![Download](https://api.bintray.com/packages/imgix/maven/imgix-java/images/download.svg) ](https://bintray.com/imgix/maven/imgix-java/_latestVersion)
```

Comment:
We can safely remove this now or wait until May 1st when all Bintray accounts are deactivated.